src/carplay.cs

```diff
@@ -785,8 +805,15 @@ interface CPWindow {
 
 		[Export ("mapButtonSafeAreaLayoutGuide")]
 		UILayoutGuide MapButtonSafeAreaLayoutGuide { get; }
+
+#if false
+// Apple is trying to use CPTemplateApplicationScene but it is not exposed. FB6231565
```

Comment:
This needs to be updated w/ the radar #, right now it's the Apple bug report #.

Reply:
yes, we now track them in https://github.com/xamarin/maccore/issues?q=is%3Aissue+is%3Aopen+label%3Aapple-feedback

Reply:
Added maccore issue #, let me know if this is the correct format!